WebServices/AzureNotificationHub/NotificationHubSample/NotificationHubSample.Android/MainActivity.cs

```diff
@@ -24,18 +18,15 @@ protected override void OnCreate(Bundle savedInstanceState)
 
             base.OnCreate(savedInstanceState);
 
-            Xamarin.Essentials.Platform.Init(this, savedInstanceState);
             global::Xamarin.Forms.Forms.Init(this, savedInstanceState);
             LoadApplication(new App());
 
-            IsPlayServiceAvailable();
-            CreateNotificationChannel();
-        }
-        public override void OnRequestPermissionsResult(int requestCode, string[] permissions, [GeneratedEnum] Android.Content.PM.Permission[] grantResults)
-        {
-            Xamarin.Essentials.Platform.OnRequestPermissionsResult(requestCode, permissions, grantResults);
+            if(IsPlayServiceAvailable() == false)
```

Comment:
You really don't like leaving spacing after `if`, do you? :)

Reply:
My thumb is so tired :P